### ISSUE TITLE
CSS Classes order

### ISSUE THREAD

**Issue by scag1io** [djyde/cusdis#52]
When using hosted service on cusdis.com, there are Svelte-xxx CSS classes which did not exist with Vercel deployment.
Since, if there are multiple CSS classes, the last one wins, it would be nice to:
- avoid Svelte-xxx classes or 
- use them in first position, so it is possible to customize classes (e.g. "submit-btn") in a separate CSS file.
<img width="603" alt="Screenshot 2021-04-25 at 11 59 02" src="https://user-images.githubusercontent.com/76182936/115989306-06b7a500-a5be-11eb-91dd-7c3390a74c08.png">

**Comment by djyde** [comment]
Hi.

svelte-xxx class name is generated by Svelte compiler. I also set `emitCss: false`, so no css file would be emitted. The style is appended to `head` in runtime. 

Did you have problem in the style of the widget?

**Comment by scag1io** [comment]
Yes, I created custom classes (e.g. "submit-btn"), but they are overwritten by the style-xxx. This is the Inspect of the button.
<img width="1312" alt="Screenshot 2021-04-25 at 15 27 43" src="https://user-images.githubusercontent.com/76182936/115995262-dd594200-a5da-11eb-8b43-ac726ac5d5b0.png">

**Comment by egoist** [comment]
since the class names are already prefixed with `cusdis-`, I think it's fine to stop using svelte's scoped css, so that the css can be overridden without using `!important`

**Comment by scag1io** [comment]
As workaround, all the classes can be included in the `#cusdis_thread` id, so they will have priority on the svelte-xxx. Example:
```
#cusdis_thread {

  .cusdis-footer {
    ...
    }
}
```

Currently it works fine for me, there are issues with the pagination button but someone is working on it :)

**Comment by djyde** [comment]
> As workaround, all the classes can be included in the `#cusdis_thread` id, so they will have priority on the svelte-xxx. Example:
> 
> ```
> #cusdis_thread {
>   .cusdis-footer {
>     ...
>     }
> }
> ```
> 
> Currently it works fine for me, there are issues with the pagination button but someone is working on it :)

I just updated the widget, check it out.

**Comment by scag1io** [comment]
> I just updated the widget, check it out.

Uhm, I get the same issue: the svelte-xxx ovveride the cusdis- classes. I think that's because [they are specified in last position](https://stackoverflow.com/questions/7637482/how-do-browsers-resolve-conflicting-classes).
<img width="1311" alt="Screenshot 2021-04-26 at 12 29 58" src="https://user-images.githubusercontent.com/76182936/116070062-6da38f80-a68c-11eb-96b7-451ae1d25638.png">


By the way, if I include in the `#cusdis_thread id`, the custom classes in the CSS are correctly visible even if the order is the same:
<img width="1305" alt="Screenshot 2021-04-26 at 12 40 28" src="https://user-images.githubusercontent.com/76182936/116070287-b78c7580-a68c-11eb-9bf8-91ba9663417b.png">

**Comment by djyde** [comment]
yea i mean the pagination button

scag1io ***@***.***>於 2021年4月26日 週一，下午6:42寫道：

**Comment by scag1io** [comment]
> yea i mean the pagination button

That's fine, but it appears also if there is only one comment. I personally would make it visible only if there is a second page.
Also, strange bug: I added 10-12 comments and a MOD reply automatically got deleted. But this is another issue.

**Comment by candideu** [comment]
> 
> 
> As workaround, all the classes can be included in the `#cusdis_thread` id, so they will have priority on the svelte-xxx. Example:
> 
> ```
> #cusdis_thread {
>   .cusdis-footer {
>     ...
>     }
> }
> ```
> 
> Currently it works fine for me, there are issues with the pagination button but someone is working on it :)

Unfortunately, this workaround isn't working for me in the hosted version. I've tried, for example:

```
#cusdis_thread .border-gray-200 {
  border: 2px solid rgba(35, 35, 36, .78) !important;
  background-color: #CBAB6C !important;
}
```

But this doesn't work to override the existing style, which doesn't mesh well with my site's existing aesthetic.
